tests/Microsoft.Identity.Test.Unit/Microsoft.Identity.Test.Unit.csproj

```diff
@@ -15,6 +15,7 @@
       <Name>Microsoft.Identity.Client</Name>
     </ProjectReference>
     <ProjectReference Include="..\Microsoft.Identity.Test.Common\Microsoft.Identity.Test.Common.csproj" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
```

Comment:
I don't see any need to add Newtonsoft to all these projects. For json processing, we should use System.Text now. Only MSAL has to use its internal copy of newtonsoft because some of advanced op.

Reply:
This is leftover form my trying to fix the device auth tests. will be removed